### PR TITLE
Feature/restore popup

### PR DESCRIPTION
**Pull request by emmapainter**
## Summary

Restore popover component!! 

<hr/>

Restore feature is now functional. Press enter to restore text while the popover is open. Uses enter instead of click due to some issues getting the component to respond to click events - @Irmy has approved this. Undo restore works too.

<hr/>

https://user-images.githubusercontent.com/100325471/187620280-d7a5c53f-7916-4d80-a1f8-e8be2a617d05.mov

### OS

- [x] Linux
- [x] MacOS
- [ ] Windows

### CODE REVIEW

**Review by Irmy — APPROVED** [review]
Love it! Looks amazing - The enter flow actually feels really natural which is awesome.

File: src/renderer/components/Editor/TranscriptionBlock.tsx [hunk 1]
```diff
@@ -160,6 +164,8 @@ const TranscriptionBlock = ({
                       seekToWord={seekToWord}
                       submitWordEdit={submitWordEdit}
                       selectionSet={selectionSet}
+                      popoverWidth={blockWidth - 194}
```
Comment (chloebrett) [inline src/renderer/components/Editor/TranscriptionBlock.tsx:167]: What's 194? Can you make it a constant so it at least has a name

File: package.json [hunk 2]
```diff
@@ -275,6 +275,7 @@
     "query-string": "^7.1.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-merge-refs": "^2.0.1",
```
Comment (chloebrett) [inline package.json:278]: Are you using this? Can't see it anywhere

File: src/main/config.ts [hunk 3]
```diff
@@ -4,4 +4,4 @@ import { TranscriptionEngine } from './handlers/helpers/transcribeTypes';
 
 // Which transcription engine to use for any transcription.
 // This will eventually be replaced with a config file allowing the user to choose which engine they want
-export const TRANSCRIPTION_ENGINE = TranscriptionEngine.ASSEMBLYAI;
+export const TRANSCRIPTION_ENGINE = TranscriptionEngine.DUMMY;
```
Comment (chloebrett) [inline src/main/config.ts:7]: Change back

**Comment by chloebrett** [comment]
Demo looks good and code is clean, will test locally soon

**Comment by chloebrett** [comment]
@emmapainter

**Pull request merged**